files/image_config/monit/memory_checker

```diff
@@ -33,6 +34,15 @@ from swsscommon import swsscommon
 EVENTS_PUBLISHER_SOURCE = "sonic-events-host"
 EVENTS_PUBLISHER_TAG = "mem-threshold-exceeded"
 
+CGROUP_DOCKER_MEMORY_DIR = "/sys/fs/cgroup/memory/docker/"
+
+# Define common error codes
+ERROR_CONTAINER_ID_NOT_FOUND = "[memory_checker] Failed to get container ID of '{}'! Exiting ..."
+ERROR_CGROUP_MEMORY_USAGE_NOT_FOUND = "[memory_checker] cgroup memory usage file '{}' of container '{}' does not exist on device! Exiting ..."
+ERROR_CONTAINER_MEMORY_USAGE_NOT_FOUND = "[memory_checker] Failed to get the memory usage of container '{}'! Exiting ..."
+ERROR_CONTAINER_CACHE_USAGE_NOT_FOUND = "[memory_checker] Failed to get the cache usage of container '{}'! Exiting ..."
+ERROR_CGROUP_MEMORY_STATS_NOT_FOUND = "[memory_checker] cgroup memory statistics file '{}' of container '{}' does not exist on device! Exiting ..."
+
 def get_command_result(command):
```

Comment:
Is it possible to add retry mechanism in case timing issue? Like below

```python
import time

def get_command_result_with_retries(command, retries=3, delay=5):
    for attempt in range(retries):
        try:
            return get_command_result(command)
        except Exception as e:
            if attempt < retries - 1:
                time.sleep(delay)
            else:
                syslog.syslog(syslog.LOG_ERR, f"[memory_checker] Command '{command}' failed after {retries} attempts. Error: {e}")
                sys.exit(2)
```

Reply:
There is already retry used in monit config, refer https://github.com/FengPan-Frank/sonic-buildimage/blob/d0d8b9f84083e545857749605ea7cf9a342a1f39/dockers/docker-sonic-telemetry/base_image_files/monit_telemetry#L5